data/ua-update.json.in

```diff
@@ -100,5 +100,6 @@
   "google.at": "Mozilla/5.0 (Android 8.1.0; U; Mobile; rv:60.9) Gecko/60.9 SailfishBrowser/1.0",
   "twitter.com": "Linux#Android 8.1.0",
   "apkpure.com": "Linux#Android 8.1.0",
-  "zingmp3.vn": "Linux#Android 8.1.0"
+  "zingmp3.vn": "Linux#Android 8.1.0",
+  "vk.com": "Mozilla/5.0 (Sailfish 3.0; Mobile; rv:60.9) Gecko/60.9 Firefox/60.9 SailfishBrowser/1.0"
```

Comment:
```suggestion
  "vk.com": "Linux; U# "
```
Simpler this way as you're only dropping "Linux; U".

Reply:
Indeed...